role_scripts/15/primary/postgresql.conf

```diff
@@ -0,0 +1,809 @@
+# -----------------------------
```

Comment:
update this file with latest config file by postgres https://github.com/postgres/postgres/blob/REL_15_STABLE/src/backend/utils/misc/postgresql.conf.sample

Reply:
I brought it from the master. will it work?